src/completions/Find.ts

```diff
@@ -1,116 +0,0 @@
-import { activeTabId } from "@src/lib/webext"
```

Comment:
This file was not used but broke after new types were propagated, so I just removed it instead of fixing it